core/domain/html_cleaner.py

```diff
@@ -405,7 +450,38 @@ def convert_to_text_angular(html_data):
     # html strings they are stored as <br>. Since both of these
     # should match and <br> and <br/> have same working,
     # so the tag has to be replaced in this way.
-    return str(soup).replace('<br/>', '<br>')
+    return unicode(soup).replace('<br/>', '<br>')
+
+
+def tag_content_to_textangular(html_data):
+    """This functions converts content with tabs and collapsible
+    components to textangular format.
+
+    Args:
+        html_data: str. The HTML string whose content is to be converted.
+
+    Returns:
+        str. The HTML string with converted content within tag.
+    """
+    soup = bs4.BeautifulSoup(html_data.encode('utf-8'), 'html.parser')
+
+    for collapsible in soup.findAll('oppia-noninteractive-collapsible'):
+        content_html = html_unescaper(collapsible['content-with-value'])
+        # This is done to ensure that &quot; is not escaped as &quot;
+        # is not escaped as &amp;amp;quot;.
```

Comment:
Comment does not make sense (has some extra words?) Please rewrite.

Reply:
Fixed this!